config/webhook/manifests.yaml

```diff
@@ -1,32 +1,3 @@
-
----
-apiVersion: admissionregistration.k8s.io/v1
```

Comment:
This is automatically generated, and kubebuilder somehow decided it's not needed to have Mutating webhook.

I tested and conversion still works.

If you think that it should stay there let me know, but FYI I tried very hard to keep mutating webhook here and failed on endpoint errors :D

Reply:
I think we should be fine with removing the mutating webhook, but @jsztuka please double check this.

Reply:
Mutating webhook should be capable of working properly even without manifests, wo yes, this should not interfere with its current behavior.